Extensions/audio_plus.js

```diff
@@ -181,26 +292,74 @@ XKit.extensions.audio_plus = {
 	check_pop_out: function() {
 		var audio_plus = XKit.extensions.audio_plus;
 		audio_plus.scroll_waiting = false;
-
-		var pause_icons = document.querySelectorAll(".post_media .audio-player .icon_pause");
-		if (pause_icons.length === 0) {
-			return;
+		
+		// Arbitrarily select the first if there are multiple
+		var pause_icon = document.querySelectorAll(".post_media .audio-player .icon_pause")[0];
+		if (pause_icon) {
+			audio_plus.show_pop_out(pause_icon);
 		}
+	},
 
-		// Arbitrarily select the first if there are multiple
-		var player = audio_plus.audio_player_of_element(pause_icons[0]);
+	show_pop_out: function(pause_icon) {
+		var player = this.audio_player_of_element(pause_icon);
 		var player_bounds = player.getBoundingClientRect();
 
 		// If not completely off the screen
 		if (player_bounds.top > -player_bounds.height) {
 			return;
 		}
 
-		audio_plus.current_player = player;
-		audio_plus.pop_out_controls.classList.add("showing");
-		audio_plus.pop_out_controls.classList.add("playing");
+		//show progress in popout container
+		var progress = player.querySelector(".progress");
+		var config = {attributes: true};
+		this.progress_observer.observe(progress, config);
+		this.icon_observer.observe(pause_icon, config);
+
+		if (player.querySelector(".track-name").innerHTML != "") {
+			this.pop_out_controls_track_name.innerHTML = player.querySelector(".track-name").innerHTML;
+		} else {
+			this.pop_out_controls_track_name.innerHTML = "Listen";
+		}
+		this.pop_out_controls_track_artist.innerHTML = player.querySelector(".track-artist").innerHTML;
+
+		this.current_player = player;
+		this.pop_out_controls.classList.add("showing");
+		this.pop_out_controls.classList.add("playing");
+		var ppIcon = this.pop_out_controls.querySelector('.play-pause').querySelector('.icon');
```

Comment:
could just use `".play-pause .icon"` in a single selector?

Reply:
Just noticed this happened in 2 other places as well, I'll fix them and re-commit ^__^